mozilla/blurts-server
# Remove Plus users' manual removals' special-casing

# References: 
Jira: MNTOR-3172
Figma: https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=11165-1860&mode=design&t=s1UG53aIWQeHAz2A-4




# Description

Apparently we'll be sending them the same monthly activity emails
as Plus users who have not manually resolved anything.

Note that I also fixed a bug where I forgot to include the generic email styles with other emails than the monthly activity email with manual resolutions. (Ironically, I called out that I was afraid for forgetting that in the [PR where I added the styles](https://github.com/mozilla/blurts-server/pull/4475) - only to do so in the very PR I called that out. "I suppose in practice people will start by copy-pasting an existing template" - except I didn't update that template :upside_down_face: )

# How to test

I left in [the story with manual resolutions](https://deploy-preview-4534--fx-monitor-storybook.netlify.app/?path=/story/emails-monthly-activity--monthly-activity-email-plus-with-manual-story) in Storybook, so you can compare that with [the one without](https://deploy-preview-4534--fx-monitor-storybook.netlify.app/?path=/story/emails-monthly-activity--monthly-activity-email-plus-without-manual-story) and see that they're the same.

# Checklist (Definition of Done)
